File: Cadastro2/app/src/main/java/com/example/aluno/cadastro2/MainActivity.java
package com.example.aluno.cadastro2;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.ArrayAdapter;
import android.widget.Button;
import android.widget.ListView;

import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ArrayList lista = new ArrayList<String>();

        lista.add("Primeira Linha..");
        lista.add("Segunda Linha..");
        lista.add("Terceira Linha..");
        lista.add("Quarta Linha..");

        ListView lv = (ListView) findViewById(R.id.listview);

        lv.setAdapter(
                new ArrayAdapter<String>
                (this,
                 android.R.layout.simple_list_item_1,
                 lista)
        );
    }
}

File: Cadastro/app/src/main/java/com/example/aluno/cadastro/MainActivity.kt
package com.example.aluno.cadastro

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnConfirmar: Button = findViewById<Button>(R.id.btnConfirmar)
        val btnLimpar: Button = findViewById<Button>(R.id.btnLimpar)

        btnLimpar.setOnClickListener {
            val nomeText: EditText = findViewById<EditText>(R.id.nome)
            val emailText: EditText = findViewById<EditText>(R.id.email)
            val loginText: EditText = findViewById<EditText>(R.id.login)
            val senhaText: EditText = findViewById<EditText>(R.id.senha)



        }
    }
}
